addOns/pscanrules/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 - The Modern App Detection scan rule now ignores non-HTML files (Issue 7617).
+- The CSP scan rule now alerts when the Content-Security-Policy header is missing, and when the obsolete X-Content-Security-Policy or X-WebKit-CSP are found (Issue 7653).
```

Comment:
```suggestion
- The CSP Missing scan rule now alerts when the Content-Security-Policy header is missing, and when the obsolete X-Content-Security-Policy or X-WebKit-CSP are found (Issue 7653).
```

Needs to include "Missing" because there is another rule in this add-on that actually does assess CSP specifically.